src/libs/fetch.ts

```diff
@@ -2,7 +2,9 @@ async function request<TResponse>(
   url: string,
   config: RequestInit = {}
 ): Promise<TResponse> {
-  const res = await fetch(url, config);
+  const baseUrl =
+    (process.env.NEXT_PUBLIC_BASE_URL ?? "https://api-alpha.42world.kr") + url;
+  const res = await fetch(baseUrl, config);
```

Comment:
이제 위에 인자 url이 아니라 path로 받는게 좀더 어울릴거같아요
```suggestion
  const baseUrl = process.env.NEXT_PUBLIC_BASE_URL ?? "https://api-alpha.42world.kr"
  const res = await fetch(`baseUrl + path`, config);
```

이런느낌은 어떠심까

Reply:
인정